docs/TriageProcess.md

```diff
@@ -0,0 +1,61 @@
+
+### Information Gathering
+In this phase we instruct the user on how to collect the appropriate diagnostics and see if they are able to address the issue with that additional information.  When we need user input we will mark the issue with `Needs Author Feedback` label. Issues in this phase may be closed automatically if we do not receive timely responses, they often do not provide enough information for us to investigate further.
+We'll try to respond quickly to such issues (within days). If a user has collected all of the relevant diagnostics and the issue is still not apparent, then we will consider it for investigation by the team.
+
+### Feature requests
+As soon as we identify an issue represents an ask for a new feature, we will label the issue with the `enhancement` label.
+Most of the time, we will automatically move these issues into the `Next Sprint Planning` milestone for further review during the [next sprint planning meeting](#milestone-planning).
+If we think the feature request is not aligned with our goals, we may choose to close it immediately.
+In some situations, however, we may choose to collect more feedback before acting on the issue. In these situations we will move the issue in the `Backlog` so that we can review it during the [release planning](#release-planning).
+
+### Bug reports
+If it's immediately clear, that the issue is related to a bug in the framework, we will apply the `bug` label to the issue.
+
+At this point, we will try to make a call regarding it's impact and severity. If the issue is critical, we may choose to include it in our current milestone for immediate handling or potentially patching.
+If the bug is relatively high impact, we will move the issue into the `Next Sprint Planning` milestone to review during our [next sprint planning](#milestone-planning) meeting.
+If the impact is unclear or the is a very corner case scenario, we may move it to the `Backlog` milestone to further evaluate the impact by reviewing customer upvotes / comments at a later time.
+
+### Investigations
+In many situations it's not immediately clear whether a specific issue repoted is a bug or not. To be certain, the team will need to spend time to investigate it before making a call regarding the faith of the issue. In these situations we will apply the `investigate` label to the issue.
+In many situations, these issues turn out to be a result of some kind of misconfiguration in the user code.
+In some rare situations, however, these turn out to be caused by very impactful issues. So we will make a call during the triage whether we need to immediately investigate certain issues or not.
+If not, we will move the investigation to the `Next Sprint Planning` to review during the [next sprint planning meeting](#milestone-planning).
```

Comment:
Customers that need help with urgent investigations should contact Microsoft Support (link).

Reply:
Added a note here on [line 6](https://github.com/dotnet/aspnetcore/pull/22393/files#diff-42b7e1e1a33ad3f1147d9d88618f0284R6]